mission_yaml/SEA45_M48.yml

```diff
@@ -154,7 +155,7 @@ netcdf_variables:
     units:        degrees
     coordinates:  time depth latitude longitude
 
-  # data parameters
+# data parameters
   conductivity:
```

Comment:
This is a god size for a PR. You could consider moving to a different text editor though, or avoid adding lines that don't have real changes, like this comment line with indentation altered. It can make looking back through the git history quite confusing.